charts/tfy-configs/files/workbench-images.yaml

```diff
@@ -41,7 +41,7 @@ images:
           image_uri: 'asia-docker.pkg.dev/production-01-407505/tfy-docker-as/jupyter:0.3.3-py3.11.10-sudo'
       - match: []
         spec:
-          image_uri: 'public.ecr.aws/truefoundrycloud/jupyter:0.3.3-py3.11.10-sudo'
+          image_uri: 'tfy.jfrog.io/tfy-images/jupyter:0.3.4-py3.11.10-sudo'
```

Comment:
We also need to bump tags in `*.pkg.dev/production-01-407505/` in this file

Reply:
am I updating it to `*.pkg.dev/production-01-407506/`